packaging/snapd.mk

```diff
@@ -72,8 +72,9 @@ $(builddir)/snap: GO_TAGS += nomanagers
 $(builddir)/snap $(builddir)/snap-seccomp $(builddir)/snapd-apparmor:
 	go build -o $@ $(if $(GO_TAGS),-tags "$(GO_TAGS)") \
 		-buildmode=pie \
-		-ldflags="-w -B 0x$$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" \
```

Comment:
Is the drop to -w accidental? If not can you split it out to a sparate commit and explain what's going on please.

Reply:
yeah, I forgot to commit that bit. I moved it out of snapd.mk so that respective distros can set whatever is needed (disabling debug info felt like a thing a distro may want to have a say in).